.gitignore

```diff
@@ -1,4 +1,5 @@
 **/.idea
 aws-services-*.brokerpak
 cloud-service-broker
+.aws-terraform-tests
 
```

Comment:
praise: thank you very much for adding the binary to the `.gitignore` file. You may want to add the `terraform-tests` folder to the `.cfignore` file

Reply:
Good catch!